packages/libs/react-components/src/components/Input/styles.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+// TODO: Remove this when this issue is resolved: https://github.com/kadena-community/kadena.js/issues/201
+
+import { styled } from '../../styles';
+
+export const styleVariant = {} as const;
```

Comment:
I assume this just needed to be cleaned up, but I think a valuable variant could be status